File: 1. Beginner/3145/3145.scala
object Main {
    def main(args: Array[String]) {
        val input = scala.io.StdIn.readLine()
        val Array(n, x) = input.split(" ").map(_.toDouble)
        val r = x / (n + 2)
        println(f"$r%.2f")
    }
}

File: 1. Beginner/3145/3145.swift
import Foundation

if let input = readLine() {
    let components = input.split(separator: " ").compactMap { Int($0) }
    if components.count == 2, let n = components.first, let x = components.last {
        print(String(format: "%.2f", Double(x) / Double(n + 2)))
    }
}